examples/incremental_learning/helmet_detection/README.md

```diff
@@ -100,7 +100,21 @@ EOF
 
 Start The Incremental Learning Job  
 
+* incremental learning supports hot model updates and cold model updates. Job support 
+cold model updates default. If you want to use hot model updates, please to set environments of the container:    
+
+```yaml
```

Comment:
As discussed, the environment variable interface for users is unsuitable, better to move it into top of spec, like:
```yaml
deploySpec:
    model:
         hotUpdateEnabled: true
```

Reply:
ok, refer to https://github.com/kubeedge/sedna/issues/104#issuecomment-901069468